components/combobox/combobox.jsx

```diff
@@ -254,6 +254,40 @@ const propTypes = {
 	 * Changes styles of the input. Currently `entity` is not supported. _Tested with snapshot testing._
 	 */
 	variant: PropTypes.oneOf(['base', 'inline-listbox', 'readonly']),
+	/**
+	 * Default value of input. Provide uncontroled behaviour
+	 */
+	defaultValue: PropTypes.string,
```

Comment:
Should we call this `defaultInputValue` ?

Reply:
Can this be added in the input prop?